### PR TITLE
Create an example pod for AWS EBS

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: Sami Wagiaalla swagiaal@redhat.com

### CODE REVIEW

**Comment by justinsb** [comment]
This looks great - thank you!  I think you may want to specify that it should be in the same AZ as your cluster, as otherwise it won't work, but this is a great improvement - people have been asking for docs on this and I've been too buried to get to it!

**Comment by swagiaal** [comment]
@justinsb thanks for taking a look. I have added a comment about region like you mentioned.

File: examples/aws_ebs/aws-ebs-web.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1beta3
```
Comment (krousey) [inline examples/aws_ebs/aws-ebs-web.yaml]: Can you make this v1? We are in the middle (I think almost complete) of converting all of our examples to v1.

#9286

Comment (krousey) [inline examples/aws_ebs/aws-ebs-web.yaml]: I'm fairly certain nothing in this spec needs to change for v1.

File: examples/aws_ebs/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+This is a simple web server pod which serves HTML from an AWS EBS
+volume.
+
+Create a volume in the same region as your node add your volume
+information in the pod description file aws-ebs-web.yaml then create
+the pod:
+
+  $ kubectl create -f aws-ebs-web.yaml
```
Comment (krousey) [inline examples/aws_ebs/README.md]: Could you wrap the shell stuff in Github's fenced code blocks? E.G.

<pre><code>```shell
$ kubectl create -f aws-ebs-web.yaml
```</code></pre>

File: examples/aws_ebs/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Add some data to the volume if is empty:
+
+  $ echo  "Hello World" >& /var/lib/kubelet/plugins/kubernetes.io/aws-ebs/mounts/aws/{Region}/{Volume ID}/index.html
+
+You should now be able to query your web server:
+
+  $ curl <Pod IP address>
+  $ Hello World
```
Comment (krousey) [inline examples/aws_ebs/README.md]: We require analytic links at the bottom of our READMEs. Shippable is currently failing this verification on your PR. Running hack/run-gendocs.sh should do that.

**Comment by swagiaal** [comment]
@krousey I have made the changes suggested.

**Comment by krousey** [comment]
Thanks so much for making those changes!

**Comment by swagiaal** [comment]
@krousey thank you for reviewing :)

**Pull request merged**